helm/alfresco-content-services/values.yaml

```diff
@@ -542,6 +542,11 @@ activemq:
   # -- Possibility to choose Node for pod, with a key-value pair label
   # e.g {"kubernetes.io/hostname": multinode-demo-m02}
   nodeSelector: {}
+  adminUser:
+    # -- Default username for the embedded broker admin user
+    user: admin
+    # -- Default password for the embedded broker admin user
+    password: admin
```

Comment:
We're already using a `messageBroker` data object for clients. I think we should not have different config elements for clients and services. 
Either we should create a secret as part of the main charts and use it as an `existingSecret`, or set credentials in values and find a way to share them across the other components.
At first sight the former looks like a better approach to me but I'm not sure this is really what best suits a reference chart